test/is.allow.token.test.ts

```diff
@@ -0,0 +1,23 @@
+import { RelayingServices } from '../src';
```

Comment:
same goes for this file. let's put all allow token tests in one file and then make clear what the difference is between each of them

Reply:
I think that there are 2 separate functions and need separate test.